WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardRemoteEntity.swift

```diff
@@ -75,10 +76,24 @@ extension BlogDashboardRemoteEntity {
             let icon: String?
         }
 
+        struct ImageSize: Decodable, Hashable {
+            let width: Int?
+            let height: Int?
```

Comment:
[Suggestion] I believe the type of these properties can be `CGFloat` even if the Backend returns them as `Int`.

Reply:
I see your point, however I would rather match the backend type unless it is an enum if that's okay with you.